docs/scanners/brakeman.md

```diff
@@ -5,10 +5,46 @@ The [Brakeman Scanner](http://brakemanscanner.org/) is a static analysis tool th
 ## Configuration
 
 Brakeman's configuration is complex and it parses its own `brakeman.ignore` file. Salus will let Brakeman reuse this file if it's present. To create one, you can use `brakeman -I` which lets you run an interactive scan.
+For more information on Brakeman configs, see [Brakeman Options](https://brakemanscanner.org/docs/options/)
 
 ## Configuration
 ```yaml
   scanner_configs:
     Brakeman:
-    - path: path/to/rails/app             # default -  looks for project_root/app
+        c: "path/to/config" # Config file for brakeman path, anything in the command line config here will override the brakeman config file. By default it will look for a config in: ./config/brakeman.yml, ~/.brakeman/config.yml, and /etc/brakeman/config.yml
+        A: true # When true, Brakeman runs all checks, off by default
```

Comment:
For these single letter options, is it possible to use a more verbose name that is a bit more intuitive? 

If its something we make our own translation for like "`config` is the `c` brakeman config option which defines the config file for brakeman patch..."

Reply:
That would make sense, but then we'd fragment from the way the scanner works, so we'd vary from brakeman's documentation.

Reply:
Actually, let's run with that. I'll make that change